src/components/user-table/body/user-table-row.tsx

```diff
@@ -94,7 +94,13 @@ const UtkastStatusData = (props: { status: UtkastStatus }) => {
 
 	return (
 		<span role="cell" className={'status'}>
-			<StatusIkon className={ikonFarge} aria-label={ariaLabel} role="img" focusable="false" />
+			<StatusIkon
+				className={ikonFarge}
+				aria-label={ariaLabel}
+				role="img"
+				focusable="false"
+				style={{ fontSize: '1.5rem', color: ikonFarge }}
```

Comment:
Hva tenker du om å fjerne `style={{ fontSize: '1.5rem', color: ikonFarge }}` og heller legge dette til i LESS-klassene for de ulike elementene? Tenker det kan være greit å ha styling i LESS og ikke inline slik at all styling er "ett" sted.